src/xb-database-manager.c

```diff
@@ -24,13 +24,15 @@
 #define QUERY_PARAM_COLLAPSE_KEY "collapse"
 #define QUERY_PARAM_CUTOFF "cutoff"
 #define QUERY_PARAM_DEFAULT_OP "defaultOp"
-#define QUERY_PARAM_LIMIT "limit"
+#define QUERY_PARAM_FLAGS "flags"
 #define QUERY_PARAM_LANG "lang"
+#define QUERY_PARAM_LIMIT "limit"
 #define QUERY_PARAM_MATCH_ALL "matchAll"
 #define QUERY_PARAM_OFFSET "offset"
 #define QUERY_PARAM_ORDER "order"
 #define QUERY_PARAM_QUERYSTR "q"
 #define QUERY_PARAM_SORT_BY "sortBy"
+#define QUERY_PARAM_FLAGS "flags"
```

Comment:
nitpick: This is added twice, I think you're missing a delete in the second commit